content/articles/rest-api-with-golang-without-any-frameworks-only-using-native-modules/index.md

```diff
@@ -0,0 +1,421 @@
+Go is a fast-growing language among developers. Its popularity is still growing and unshaken. Golang ecosystem has a tone of native libraries. However, it has third-party libraries that developers can still choose to use.
+
+When creating Rest API with Go, you might choose to use libraries such as Gorm, Go fiber, Gin, fast HTTP, etc. On the same, you can still combine this with other third-party libraries to come up with the API of your choice.
+
+However, due to the many libraries that the Go ecosystem has, you can still choose native libraries that you don't have to download and build your APIs. In this guide, we will build a complete and functional REST API. This API will only use native libraries that Golang has.
```

Comment:
These three paragraphs are somewhat confusing. The meaning doesn't come out clearly.